### PR TITLE
51660: added missing permissions. 

### PR DESCRIPTION
**Pull request by bubudrc**

### CODE REVIEW

File: AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs [hunk 1]
```diff
@@ -301,14 +311,20 @@ static NSUrl fetchMovieUrl ()
...
+			if (movieURL == null)
+			{
+				UIAlertView alertView = new UIAlertView(null, "Could not find any movies in assets library to use as sample content.", null, "OK", null);
+				alertView.Show();
 			}
 
 			return movieURL;
 		}
 
+		bool isMovieURLValid()
```
Comment (olegoid) [inline AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs]: Methods names should start with uppercase characters. Check this document for details:
http://www.mono-project.com/community/contributing/coding-guidelines/

File: AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs [hunk 2]
```diff
@@ -301,14 +311,20 @@ static NSUrl fetchMovieUrl ()
...
+		{
+			return (movieUrl != null);
```
Comment (olegoid) [inline AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs]: I believe that braces here are not required.

File: AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs [hunk 3]
```diff
@@ -91,7 +100,8 @@ public override void ViewDidLoad ()
 
 			// Start observing player's status.
 			// TODO: https://trello.com/c/SeE2FBoN
-			Player.AddObserver (this, "status", NSKeyValueObservingOptions.Initial | NSKeyValueObservingOptions.New, IntPtr.Zero);
+			if (Player != null)
```
Comment (olegoid) [inline AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs]: Here you can use nullability operator `Player.?...` to make the code more compact.

File: AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs [hunk 4]
```diff
@@ -23,7 +23,16 @@ public class MainViewController : UIViewController
 
 		public AVPlayer Player {
 			get {
-				return player = player ?? new AVPlayer (movieUrl);
+
+				if (player == null)
```
Comment (olegoid) [inline AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs]: Please combine `player == null` and `IsMovieURLValid` in one `if` and update curly braces position in accordance with Mono coding guidelines:
http://www.mono-project.com/community/contributing/coding-guidelines/

File: AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs [hunk 5]
```diff
@@ -301,14 +311,20 @@ static NSUrl fetchMovieUrl ()
 
 			waitHandle.WaitOne ();
 
-			if (movieURL == null) {
-				UIAlertView alertView = new UIAlertView (null, "Could not find any movies in assets library to use as sample content.", null, "OK", null);
-				alertView.Show ();
+			if (movieURL == null)
```
Comment (olegoid) [inline AudioTapProcessor/AudioTapProcessor/MainScreen/MainViewController.cs]: Please update position of curly braces and add missing spaces.

**Comment by bubudrc** [comment]
DONE

**Comment by olegoid** [comment]
LGTM

**Pull request merged**